tests/lib/rule-tester/rule-tester.js

```diff
@@ -2122,13 +2191,27 @@ describe("RuleTester", () => {
                     invalid: [{
                         code: "var foo;",
                         errors: [{
+                            message: "Bad var.",
                             suggestions: [{
                                 messageId: "this-does-not-exist"
                             }]
                         }]
                     }]
                 });
-            }, "Error should have an array of suggestions. Instead received \"undefined\" on error with message: \"Bad var.\"");
+            }, 'Error should have no suggestions on error with message: "Bad var."');
```

Comment:
This seems wrong. The test case expects suggestions but the actual lint error doesn't have suggestions, so the previous message "Error should have an array of suggestions" was correct.